### ISSUE TITLE
Route mpi.dask.org to dask-mpi.readthedocs.io

### ISSUE THREAD

**Issue by mrocklin** [dask/dask-mpi#13]
@aterrel would it be possible to have mpi.dask.org route to dask-mpi.readthedocs.io ?

**Comment by mrocklin** [comment]
pinging @aterrel again

(also, is there anyone else around that has access to the DNS records in case we want to diffuse responsibility?)

**Comment by aterrel** [comment]
sorry about the delay. Github is not a great way to get ahold of me, better ways:

- email web@numfocus.org (where several people can get your email)
- email info@numfocus.org
- email andy.terrel@gmail.com
- ping me on NF Slack
- ping me on my cell (which you have)

I've tried getting people to manage this stuff but it seems only martey and me.

I've pointed dns for mpi.dask.org to readthedocs. Now someone on the dask side need to point
